enterprise/kubernetes.md

```diff
@@ -0,0 +1,108 @@
+# Installing Private Packagist Enterprise on Kubernetes
+
+Private Packagist Enterprise can run on a Kubernetes cluster. To do that, Private Packagist Enterprise uses Replicated.
+
+Replicated is the application which will provide you with a management interface to your Private Packagist Enterprise installation and allow you to monitor the task and perform maintenance operations such as backups or updates.
+
+## General requirements
+
+1. License Key File (file extension .rli) Don't have one yet? [Sign up for a free trial license!](https://packagist.com/enterprise)
+2. One (sub-)domain to operate the web interface, e.g. packagist.myintranet.com
+3. One (sub-)domain to operate the composer repository, e.g. repo.packagist.myintranet.com or packagist-repo.myintranet.com
+4. An SSL certificate valid for both chosen domains
+5. An SMTP server or a GMail account for Private Packagist Enterprise to send email
+
+## Options
+
+There are two options for running Private Packagist Enterprise on a Kubernetes cluster:
+
+* **Start from scratch.** Installing Replicated Kubernetes on a Linux server and running Private Packagist Enterprise there
+* **Use an existing Kubernetes cluster.** Installing Replicated on your existing Kubernetes cluster and running Private Packagist Enterprise there
+
+## Installing Private Packagist Enterprise from scratch
+
+### Requirements
+
+* A Linux Server
+  * OS
+    * Ubuntu 16.04.3 - 16.04.5
+    * Red Hat Enterprise Linux 7.4 - 7.5
+    * CentOS 7.4 - 7.5
```

Comment:
Where did this list come from?

Reply:
Copied it from here: https://help.replicated.com/docs/kubernetes/customer-installations/supported-operating-systems/

Reply:
Ah thanks, that appears to be for the "easy install script" only. Other distributions and versions etc. are supported for manual installation, too. So I'd rather not include the list here specifically.